lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -157,6 +111,7 @@
     padding: ($baseline/4) ($baseline/2);
     color: $black;
     text-align: right;
+    border-radius: $forum-border-radius $forum-border-radius 0 0;
```

Comment:
That should be:

    @include border-radius($forum-border-radius, $forum-border-radius, 0, 0);